<launch>
    <!-- <arg name="my_int_param" default="1"/> -->
    <!-- <param name="my_int_param" value="$(arg my_int_param)" /> -->
    <rosparam command="load" file="$(find my_package)/config/params.yaml"/>

    <node name="talker" pkg="my_package" type="my_package_node" output="screen"/>
</launch>



